src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -1121,6 +1121,30 @@ public void testListPaymentStatusTransitions() throws Exception {
         assertThat(returnValue.getData().get(1).getToStatus(), is(equalTo(RECALLED)));
     }
 
+    @Test
+    public void testListPayments() throws Exception {
+        String functionality = "listPayments";
+        initMockServer(functionality);
+        HyperwalletPaymentListOptions options = new HyperwalletPaymentListOptions();
+        options.clientPaymentId("gv47LDuf")
+                .sortBy("test-sort-by")
+                .limit(10);
+        HyperwalletList<HyperwalletPayment> returnValue;
+        try {
+            returnValue = client.listPayments(options);
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+        assertThat(returnValue.getData().get(0).getToken(), is(equalTo("pmt-3ffb5fcc-1c98-48ce-9a6c-e4759933a037")));
```

Comment:
Can't these asserts be in the try block? None of them will execute if an exception is thrown. It saves having to declare `returnValue` outside the try block.

Reply:
I think it is following the pattern of other tests in this class and keeps the try block small, I will prefer this.

Reply:
Ok. I still think it is unnecessary extra lines but if this is the pattern/style existing tests used that's fine.